decidim-meetings/app/commands/decidim/meetings/admin/copy_meeting.rb

```diff
@@ -55,6 +55,11 @@ def copy_meeting!
             component: @meeting.component,
             registrations_enabled: @meeting.registrations_enabled,
             available_slots: @meeting.available_slots,
+            has_conciliation_service: @form.has_conciliation_service,
```

Comment:
I don't have a technical issue with this, but maybe conciliation is not the right term?

[Conciliation](https://www.wordnik.com/words/conciliation): "the action of stopping someone from being angry; placation."

Maybe `family friendly space` or `childcare service` would make more sense @decidim/product ?

Reply:
@core According with @decidim/product we decide to change the admin functionality with a jsonb field, called for example 'services_offered' and store in them the information of the services. They can add up to 4. Any help on how to do it will be helpful 😃

Reply:
I'd say the adding this limit (4) might add a lot of complexity, and since this is something only admins will use I wouldn't limit it.

Reply:
What do you propose? Are you agree to use a jsonb field with all the services with translations?  Or do you think there's a better way?

Reply:
I'm OK with using a JSONb field, but without limiting the number of services